src/docs/spec.md

```diff
@@ -362,6 +362,8 @@ This precludes spurious diffs in a git setting, which is an important concern fo
 
 Note that only metadata elements permissible in a global context (G, or L/G) can be used in the metadata-file.
 
+**Entity references**: All identifiers in a SSSOM/TSV file (both in the metadata section and in the TSV columns) MUST be in CURIE form. The use of full-length identifiers is officially not supported.
```

Comment:
## The change

Reply:
I’ve already put that change in my #362 PR. It just need to be approved and merged.

Reply:
That is good to hear, thanks! I'll close this, tehn.